tslint.json

```diff
@@ -0,0 +1,14 @@
+{
+    "extends": "tslint:recommended",
+    "rules": {
+      "max-line-length": {
+        "options": [100]
+      },
+      "member-ordering": false,
+      "no-consecutive-blank-lines": false,
+      "object-literal-sort-keys": false,
+      "ordered-imports": false,
+      "quotemark": [true, "single"],
+      "variable-name": [true, "allow-leading-underscore"]
```

Comment:
I believe having leading or trailing underscores encourages bad naming. IMO we should deny it by default. Do you have any example where it might be helpful?

Reply:
Changed this to CameCase. I hate underscores too, This config I copied and pasted some parts so i didn't fully review it.